x-pack/filebeat/input/httpjson/config/config.go

```diff
@@ -2,7 +2,7 @@
 // or more contributor license agreements. Licensed under the Elastic License;
 // you may not use this file except in compliance with the Elastic License.
 
-package httpjson
+package config
```

Comment:
do we really need a sub-package for config? This package is still tightly coupled to the httpjson package. What additional functionality does this package provide that is required outside of this (or the httpjson) package?